bower.json

```
@@ -19,27 +19,22 @@
     "tests"
   ],
   "dependencies": {
-    "alphaville-ui": "Financial-Times/alphaville-ui#^2.6.0",
-    "alphaville-marketslive-chat": "Financial-Times/alphaville-marketslive-chat#^1.0.0",
-    "webchat": "Financial-Times/webchat#^2.1.0",
+    "alphaville-ui": "#c5d9bb6",
+    "alphaville-marketslive-chat": "#upgrade-dependencies-25-11-19",
+    "webchat": "#upgrade-dependencies-28-11-19 ",
```

For when [this is released](https://github.com/Financial-Times/webchat/pull/2):
```suggestion
    "webchat": "^3.0.0",
```